### DISCUSSION TITLE
How accurate are it's timestamps vs whisperX

### DISCUSSION THREAD

**Discussion by creativeidiot123** [linto-ai/whisper-timestamped#197] [General]
These two are the most popular timestamp generators for whisper backend. However i don't know which one's better with timestamps. On paper whisper-timestamped is better but i tried both and there seemed to be some issues with timestamped timestamps. 

Anyone else who can tell me anything related to this?

**Comment by Yohan85** [comment]
Hi @ankitgurua , did you end up finding the best candidate for your use case? Which version of WhisperX are you referring to? Do you have a github link?
